ibis/pyspark/tests/test_basic.py

```diff
@@ -174,3 +174,28 @@ def test_join(client):
     )
 
     tm.assert_frame_equal(result.toPandas(), expected.toPandas())
+
+
+def test_filter(client):
```

Comment:
I think you should use pytest.mark.parametrize here. you can take a look here to have an idea:  https://github.com/ibis-project/ibis/blob/master/ibis/tests/all/test_numeric.py#L15

Reply:
Agreed, good idea.